### PR TITLE
Mute/Unmute participants from a call - CallAutomation

### PR DESCRIPTION
**Pull request by cochi2**
This PR enables Mute and Unmute operations within a call.
CFV: https://ngc.skype.net/call/0444c3c3-d783-44ac-999b-5efbcd8278a2/

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/communication/Azure.Communication.CallAutomation/src/Models/MuteAllParticipantsRequestInternal.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallAutomation.Models
```
Comment (minwoolee-msft) [inline sdk/communication/Azure.Communication.CallAutomation/src/Models/MuteAllParticipantsRequestInternal.cs:6]: Namespace needs to be updated to align: Azure.Communication.CallAutomation

File: sdk/communication/Azure.Communication.CallAutomation/src/Models/MuteParticipantRequestInternal.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallAutomation.Models
```
Comment (minwoolee-msft) [inline sdk/communication/Azure.Communication.CallAutomation/src/Models/MuteParticipantRequestInternal.cs:6]: Same here
Azure.Communication.CallAutomation

File: sdk/communication/Azure.Communication.CallAutomation/src/CallConnection.cs [hunk 3]
```diff
@@ -625,5 +624,445 @@ public virtual CallMedia GetCallMedia()
...
+            {
+                OperationContext = operationalContext,
+            };
+            return UnmuteParticipant(options, cancellationToken);
+        }
+
+        /// <summary>
+        /// Unmute a single participant from the call.
+        /// </summary>
+        /// <param name="options"></param>
```
Comment (minwoolee-msft) [inline sdk/communication/Azure.Communication.CallAutomation/src/CallConnection.cs:703]: Should we add descriptions to these input parameters?

File: sdk/communication/Azure.Communication.CallAutomation/src/CallConnection.cs [hunk 4]
```diff
@@ -4,12 +4,11 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Net.Http;
-using System.Net;
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
 using Azure.Core.Pipeline;
+using Azure.Communication.CallAutomation.Models;
```
Comment (minwoolee-msft) [inline sdk/communication/Azure.Communication.CallAutomation/src/CallConnection.cs:11]: I think we can remove this

**Review by minwoolee-msft — APPROVED** [review]

**Review by bharat-kalyan-namburi — APPROVED** [review]

**Pull request merged**